### PR TITLE
docs(api): refactor-building-block-commands

### PR DESCRIPTION
**Pull request by jwwojak**
# Overview

This is the parent PR for revisions and updates to the API doc, [Building Block Commands](https://docs.opentrons.com/v2/new_atomic_commands.html) (`new-atomic-commands.rst`).

This is the parent PR for multiple changes. We're working this way because the original doc is very long. It's easier to manage the writing and reviews if we break the project up into a few smaller PRs. See [this Figma file](https://www.figma.com/file/WsKj7gklIgDx68DkzegdaN/Commands-PAPI-docs-merge-strategies?type=whiteboard&node-id=0%3A1&t=PrAoIpzPVTdzpnY3-1) for the strategy at work here.

Earlier effort was https://github.com/Opentrons/opentrons/pull/13282. This PR replaces that previous version.

# Test Plan

- Break up the revisions into multiple, smaller PRs for review.
- Merge small PRs into this main.
- Push the main PR into edge.
- The usual checks for grammar, style, accuracy, links, etc.

Sandbox: http://sandbox.docs.opentrons.com/docs-refactor-building-block-commands/v2/

# Changelog

TBD. Will update when ready.

# Review requests

TBD. Will update when ready.

# Risk assessment

Low, documentation changes only.

### CODE REVIEW

**Comment by jwwojak** [comment]
Changing this from draft to normal PR. However, the current doc is very long. As a result, we're going to break up the sections into separate pages. This means more changes coming, but only to the structure not the content.

File: api/docs/v2/new_atomic_commands.rst [hunk 1]
```diff
@@ -1,547 +1,20 @@
...
 
-Building block, or basic, commands are the smallest individual actions that can be completed on an Opentrons robot.
-For example, the complex command ``transfer`` (see :ref:`v2-complex-commands`) executes a series of ``pick_up_tip()``, ``aspirate()``, ``dispense()`` and ``drop_tip()`` basic commands.
+.. toctree::
+    basic_commands/pipette_tips
+    basic_commands/liquids
+    basic_commands/utilities
 
-The examples in this section would be added to the following:
+Building block commands execute some of the most basic actions that your robot can complete. But basic doesn’t mean these commands lack capabilities. They perform important tasks in your protocols. They're also foundational to the :ref:`complex commands <v2-complex-commands>` that help you write and run longer, more intricate procedures.
```
Comment (ecormany) [inline api/docs/v2/new_atomic_commands.rst]: I think that using building block commands leads to "longer, more intricate" code. Complex commands are shortcuts — with some limitations! Maybe `that help you combine multiple actions into fewer lines of code.`

Comment (jwwojak) [reply api/docs/v2/new_atomic_commands.rst]: Change implemented!

File: api/docs/v2/basic_commands/pipette_tips.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+The following sections demonstrate how to use each method and include sample code. The examples used here assume that you've loaded the pipettes and labware from the basic :ref:`protocol template <protocol-template>`.
+
+Picking Up a Tip
+================
+
+To pick up a tip, call the :py:meth:`~.InstrumentContext.pick_up_tip` method without any arguments::
+    
+    pipette.pick_up_tip()
+
+This simple statement works because the variable ``tiprack_1`` in the sample protocol includes the on-deck location of the tip rack (Flex ``location="D3"``, OT-2 ``location=3``) *and* the ``pipette`` variable includes the argument ``tip_racks=[tiprack_1]``. Given this information, the robot moves to the tip rack and picks up a tip from position A1 in the rack. On subsequent calls to ``pick_up_tip()``, the robot will use the next available tip. For example::
```
Comment (ecormany) [inline api/docs/v2/basic_commands/pipette_tips.rst:20]: True Pythonistas might nitpick at the use of "variable" here, but I think it's clear enough. Let's wait until someone actually complains 😈

File: api/docs/v2/basic_commands/pipette_tips.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+If your protocol requires a lot of tips, add a second tip rack to the protocol. Then, associate it with your pipette and increase the number of repetitions in the loop. The robot will work through both racks. 
+
+First, add another tip rack to the sample protocol::
+
+    tiprack_2 = protocol.load_labware(
+        load_name="opentrons_flex_96_tiprack_1000ul",
+        location="C3"
+    )
+
+Next, revise the pipette's ``load_instrument()`` method to include the new tip rack in the ``tip_rack`` argument::
```
Comment (ecormany) [inline api/docs/v2/basic_commands/pipette_tips.rst]: The terminology here may be too far off base. We're definitely not modifying the _method_. We're modifying the code that calls the method.

Also, when you've already started with the basic template, this code isn't copy-paste-able. We're asking users to go back and modify the template. But we don't have to! `InstrumentContext.tip_racks` is a mutable property. You can just do:
```
pipette.tip_racks.append(tiprack_2)
```

File: api/docs/v2/basic_commands/pipette_tips.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    pipette.drop_tip()                   # drops tip in trash bin
+    pipette.pick_up_tip(tiprack_1['A1']) # picks up tip from rack location A1
+
+Early API versions treated returned tips as unused items. They could be picked up again without an explicit argument. For example:: 
+
+    pipette.pick_up_tip()  # picks up tip from rack location A1
+    pipette.return_tip()   # drops tip in rack location A1
+    pipette.pick_up_tip()  # picks up tip from rack location A1
+
+.. versionchanged: 2.2
```
Comment (ecormany) [inline api/docs/v2/basic_commands/pipette_tips.rst]:
```suggestion
.. versionchanged:: 2.2
```
It's an invisible comment without the double colon…which sent me looking around trying to figure out when this change was!

File: api/docs/v2/basic_commands/liquids.rst [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+    pipette.aspirate(200, plate['A1'])
+    protocol.delay(seconds=5) # pause for 5 seconds
+    pipette.aspirate(100)     # aspirate 100 µL at current position
+
+Now our pipette holds 300 µL.
+
+Aspirate by Well or Location
+----------------------------
+
+The :py:meth:`~.InstrumentContext.aspirate` method includes a ``location`` parameter that accepts either a ``Well`` or a ``Location``.
```
Comment (ecormany) [inline api/docs/v2/basic_commands/liquids.rst]: We could link these:
```suggestion
The :py:meth:`~.InstrumentContext.aspirate` method includes a ``location`` parameter that accepts either a :py:class:`.Well` or a :py:class:`~.types.Location`. 
```

File: api/docs/v2/basic_commands/liquids.rst [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+
+Aspirate by Well or Location
+----------------------------
+
+The :py:meth:`~.InstrumentContext.aspirate` method includes a ``location`` parameter that accepts either a ``Well`` or a ``Location``. 
+
+If you specify a well, like ``plate['A1']``, the pipette will aspirate from a default position 1 mm above the bottom center of that well. To change the default clearance, first set the ``aspirate`` attribute of :py:obj:`.well_bottom_clearance`:: 
+
+    pipette.pick_up_tip
+    pipette.well_bottom_clearance.aspirate = 2 # tip is 2 mm above well bottom
```
Comment (ecormany) [inline api/docs/v2/basic_commands/liquids.rst]:
```suggestion
    pipette.well_bottom_clearance.aspirate = 2  # tip is 2 mm above well bottom
```
🧐 
There's a few more of these, if we care.

File: api/docs/v2/basic_commands/liquids.rst [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+.. code-block:: python
+    
+    pipette.dispense(100, plate['B1'])
+    protocol.delay(seconds=5) # pause for 5 seconds
+    pipette.dispense(100)     # dispense 100 µL at current position
+    
+Dispense by Well or Location
+----------------------------
+
+The :py:meth:`~.InstrumentContext.dispense` method includes a ``location`` parameter that accepts either a ``Well`` or a ``Location``.
```
Comment (ecormany) [inline api/docs/v2/basic_commands/liquids.rst]: Link the classes or don't, whichever you decide for aspirate above.

File: api/docs/v2/basic_commands/liquids.rst [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+
+To blow an extra amount of air through the pipette's tip, call the :py:meth:`.InstrumentContext.blow_out` method. You can use a specific well in a well plate or reservoir as the blowout location. If no location is specified, the pipette will blowout from its current well position::
+
+    pipette.blow_out()
+
+You can also specify a particular well as the blowout location::
+
+    pipette.blow_out(plate['B1'])
+
+Many protocols use trash bin for blowing out the pipette. You can specify the trash bin as the blowout location by using the :py:meth:`.ProtocolContext.fixed_trash` method::
```
Comment (ecormany) [inline api/docs/v2/basic_commands/liquids.rst]:
```suggestion
Many protocols use the trash bin for blowing out the pipette. You can specify the trash bin as the blowout location by using the :py:obj:`.ProtocolContext.fixed_trash` property::
```
Definitely not a method.

File: api/docs/v2/basic_commands/liquids.rst [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+
+The ``touch_tip`` feature allows the pipette to touch the edges of a well gently instead of crashing into them. It includes the ``radius`` argument. When ``radius=1`` the robot moves the centerline of the pipette’s plunger axis to the edge of a well. This means a pipette tip may sometimes touch the well wall too early, causing it to bend inwards. A smaller radius helps avoid premature wall collisions and a lower speed produces gentler motion. Different liquid droplets behave differently, so test out these parameters in a single well before performing a full protocol run.
+
+.. warning::
+    *Do not* set the ``radius`` value greater than ``1.0``. When ``radius`` is > ``1.0``, the robot will forcibly move the pipette tip across a well wall or edge. This type of aggressive movement can damage the pipette tip and the pipette.
+
+Touch Speed
+-----------
+
+Touch speed controls how fast the pipette moves in mm/s during a touch tip step. The default movement speed is 60 mm/s, the minimum is 20 mm/s, and the maximum is 80 mm/s. Calling ``touch_tip`` without any arguments moves a tip at the default speed in the current well::
```
Comment (ecormany) [inline api/docs/v2/basic_commands/liquids.rst]: Minimum of 20 here contradicts the `versionchanged` notice below. What we say here in the main text should be the minimum on the latest version of the API.

**Review by ecormany — APPROVED** [review]
With anything this long, it's always possible to find more things on subsequent reads. Some of the stuff I commented on this time around can stay as is. One or two things probably merit one more commit on this branch. But then it's time to :shipit:

File: api/docs/v2/basic_commands/pipette_tips.rst (hunk 4 above)
Comment (jwwojak) [reply api/docs/v2/basic_commands/pipette_tips.rst]: Oh, I'm missing a colon. I was wondering why the version changed note showed up as a green comment. I just thought it was something weird with the syntax coloring.

File: api/docs/v2/basic_commands/liquids.rst (hunk 9 above)
Comment (jwwojak) [reply api/docs/v2/basic_commands/liquids.rst]: Fixing now.

File: api/docs/v2/basic_commands/liquids.rst (hunk 7 above)
Comment (jwwojak) [reply api/docs/v2/basic_commands/liquids.rst]: I see what's going on. Thought the linked reference in the "Aspirate by Well" section covered it. Will update to match.

**Pull request merged**